package.json

```diff
@@ -76,7 +78,8 @@
     "tsconfig-paths": "^2.2.0",
     "typescript": "^2.4.2",
     "uuid": "^3.0.1",
-    "uuid-validate": "^0.0.2"
+    "uuid-validate": "^0.0.2",
+    "snyk": "^1.41.1"
```

Comment:
well this shouldn't be a dependency

Reply:
devDependency, I tried to run the wizard a couple of weeks ago but ran into issues around nodejs logging and its old dependencies, yesterday I merged updating all of them.

So can give this another try when we have time